chronicles-rulebook.md

```diff
@@ -2919,13 +2907,13 @@ Item production is the process by which characters are able to craft consumable
 #### Rank VII
 
 ##### Blade of Beginner’s Luck
-   >Enhanced with the Luck of a 5-Leaf Clover, this specially treated blade finds unlikely avenues and helps the wearer strike true where otherwise their blow may have been turned aside. The wielder does +5 Bonus Damage on all uses of any Critical Strike skill while wielding this Blade. But Luck is Fickle, and harder to come by as one grows older, as such, only an individual with 250 total SP or less may benefit from this weapon. This is a Passive effect.<br/>
+   >Enhanced with the Luck of a 5-Leaf Clover, this specially treated blade finds unlikely avenues and helps the wearer strike true where otherwise their blow may have been turned aside. The wielder does an extra +5 Bonus Damage while using Critical Strike skill while wielding this Blade. But Luck is Fickle, and harder to come by as one grows older, as such, only an individual with 250 total SP or less may benefit from this weapon. This is a Passive effect.<br/>
```

Comment:
Please update "Critical Strike skill" to "Critical Strike ability" since it is now a Stamina ability rather than a skill.